bin/jsonlint

```diff
@@ -38,32 +38,40 @@ if (isset($_SERVER['argc']) && $_SERVER['argc'] > 1) {
         } else if ($arg == '-h' || $arg == '--help') {
             showUsage();
         } else {
-            $file = $arg;
+            $files[] = $arg;
         }
     }
 }
 
-if ($file) {
-    if (!preg_match('{^https?://}i', $file)) {
-        if (!file_exists($file)) {
-            fwrite(STDERR, 'File not found: '.$file.PHP_EOL);
-            exit(1);
-        }
-        if (!is_readable($file)) {
-            fwrite(STDERR, 'File not readable: '.$file.PHP_EOL);
-            exit(1);
+if (!empty($files)) {
+    foreach($files as $file) {
+        if (!preg_match('{^https?://}i', $file)) {
+            if (!file_exists($file)) {
+                fwrite(STDERR, 'File not found: '.$file.PHP_EOL);
+                exit(1);
+            }
+            if (!is_readable($file)) {
+                fwrite(STDERR, 'File not readable: '.$file.PHP_EOL);
+                exit(1);
+            }
         }
     }
 } else {
     if ($contents = file_get_contents('php://stdin')) {
         lint($contents);
+        exit(0);
     } else {
         fwrite(STDERR, 'No file name or json input given'.PHP_EOL);
         exit(1);
     }
 }
 
-lintFile($file, $quiet);
+
+echo 'Checking '.count($files). " file".(count($files) > 1  ? 's' : '').PHP_EOL;
+foreach($files as $file) {
+    lintFile($file, $quiet);
```

Comment:
that looks weird to me. Linting will stop on the first failing file. Shouldn't we always process all files ?